.orchestra/config/components/openssl.yml

```diff
@@ -0,0 +1,53 @@
+#@ load("@ytt:data", "data")
```

Comment:
`Co-authored-by: Davide Depau <davide@rev.ng>`?

Reply:
I did minor changes to the file so I set you as author and me as co-author

Reply:
Ahah, OK.